itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -589,6 +590,72 @@ public void testIncrementalAdds() throws IOException {
   }
 
   @Test
+  public void testIncrementalLoadWithVariableLengthEventId() throws IOException, TException {
```

Comment:
This won't work now as INSERT generates 3 events ALTER+INSERT+ALTER. So, if we set 99 as start ID and have one insert, then insert will cross the boundary of 100. If we set start as 98, then it will be problem if in future we reduce the number of events for insert. So, I feel, it is better to keep 5 inserts for safer side.